api/src/main/java/com/redhat/ipaas/api/v1/rest/util/PaginationOptions.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.ipaas.api.v1.rest.util;
+
+public interface PaginationOptions {
+
+    int getPage();
```

Comment:
One could use checkstyle-maven-plugin for this http://checkstyle.sourceforge.net/config_javadoc.html
Of if we want to have more sophisticate metrics somethings like SonarQube (example: https://sonarqube.com/dashboard/index?id=org.jolokia%3Ajolokia or https://sonarqube.com/dashboard/index?id=io.fabric8%3Afabric8-maven-plugin-build). Really love SonarQube but can understand if people don't care much as it is quite some work to care about metrics.